### PR TITLE
Update OD TV snapshots

### PR DESCRIPTION
**Pull request by rebeccamcginn**
Resolves #NUMBER

**Overall change:**
Updating incorrect snapshots for OD TV integration tests

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by ryanmccombe — APPROVED** [review]

**Comment by jroebu14** [comment]
how was it possible to merge the work without these snapshot updates??

**Comment by rebeccamcginn** [comment]
> how was it possible to merge the work without these snapshot updates??

I _believe_ the new snapshots were lost in a merge conflict and then didn't exist as committed snapshots. If you pull latest & run the integration tests, it creates these.

File: src/integration/pages/onDemandTVPage/pashto/__snapshots__/amp.test.js.snap [hunk 1]
```diff
@@ -160,24 +160,6 @@ exports[`AMP On Demand T V Page Header Navigation link should match text and url
 
 exports[`AMP On Demand T V Page Header Navigation link should match text and url 9`] = `"کالم - /pashto/columns"`;
 
-exports[`AMP On Demand T V Page Header Navigation link should match text and url 1`] = `"کور پاڼه - /pashto"`;
```
Comment (karinathomasbbc) [inline src/integration/pages/onDemandTVPage/pashto/__snapshots__/amp.test.js.snap:163]: I don't understand why these have been reverted - they should be generated by #6282

Comment (rebeccamcginn) [reply src/integration/pages/onDemandTVPage/pashto/__snapshots__/amp.test.js.snap:163]: They have been duplicated

**Review by karinathomasbbc — APPROVED** [review]
thanks for fixing

**Comment by rebeccamcginn** [comment]
> > how was it possible to merge the work without these snapshot updates??
> 
> I _believe_ the new snapshots were lost in a merge conflict and then didn't exist as committed snapshots. If you pull latest & run the integration tests, it creates these.

Nope, this is incorrect. My original PR had the correct snapshots: https://github.com/bbc/simorgh/pull/6733

**Pull request merged**